soxr-sys/src/lib.rs

```diff
@@ -24,8 +24,8 @@ mod tests {
 
         use hound::{WavReader, WavSpec, WavWriter};
 
-        let input_wav_path = "input.wav";
-        let output_wav_path = "output.wav";
+        let input_wav_path = "../examples/play_from_disk/change-sophie.wav";
+        let output_wav_path = "test-output.wav";
```

Comment:
input.wav was missing in main - updated the test to use an existing file from examples and added the test output to gitignore @theomonnom